debezium/kubernetes/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine
+RUN apk add --no-cache curl postgresql-client tar
+RUN wget https://github.com/golang-migrate/migrate/releases/download/v4.6.2/migrate.linux-amd64.tar.gz && tar -xf migrate.linux-amd64.tar.gz
+RUN mv migrate.linux-amd64 usr/bin/migrate
+COPY . .
```

Comment:
this will copy all source code (if we are already using alpine, let's keep it slim 😅 ), but we only need  a few selected files and folders

Reply:
it makes sense this dockerfile is not multi-stage, I'm wondering why each service copy all source code although we only need few files😅